daft/runners/ray_runner.py

```diff
@@ -114,12 +118,6 @@
 except ImportError:
     _NUMPY_AVAILABLE = False
```

Comment:
We should also use the new pattern for numpy to stay consistent!

Reply:
Gotcha, done